tests/Fixtures/JsonValidator.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\Content\Tests\Fixtures;
+
+use PHPUnit\Framework\TestCase;
+
+class JsonValidator {
+
+	private TestCase $testCase;
+	private string $basePath;
+
+	public function __construct( TestCase $testCase ) {
+		$this->testCase = $testCase;
+		$this->basePath = __DIR__ . '/../../';
+	}
+
+	public function assertJsonFilesHaveMatchingKeys( string $jsonFilePathA, $jsonFilePathB ): void {
+		$this->testCase->assertEqualsCanonicalizing(
```

Comment:
The output of this comparison is really hard to read, because it shows the full content of the two arrays. Could you instead do two calls to `array_diff` with both sets of keys, expected the resulting array to equal an empty array (and a custom error message for the assertion to show what the result means)? This will show only keys that were added to one of the files but not the others.